src/GitVersionCore/Core/GitPreparer.cs

```diff
@@ -50,22 +48,17 @@ public void Prepare()
         private void PrepareInternal(bool normalizeGitDirectory, string currentBranch, bool shouldCleanUpRemotes = false)
         {
             var gitVersionOptions = options.Value;
-            if (!string.IsNullOrWhiteSpace(gitVersionOptions.RepositoryInfo.TargetUrl))
```

Comment:
I removed the dynamic repository stuff as per the comments on this PR. User can clone down their own repo and run gitversion normalise against it.

Reply:
Yes, good riddance! 😅